src/main/include/log4cxx/helpers/aprinitializer.h

```diff
@@ -44,7 +44,11 @@ class FileWatchdog;
 class APRInitializer
 {
 	public:
+#if LOG4CXX_ABI_VERSION <= 15
 		static log4cxx_time_t initialize();
+#else
+		static log4cxx_time_t getStartTime();
+#endif
```

Comment:
Instead of having a `#ifdef` here, would it make more sense to just have both methods?

To keep ABI compatibility it's okay to add methods, we just can't remove old ones unless we do an ABI bump.

Reply:
Yes, I agree in hindsight